<launch>

    <node pkg="rviz" type="rviz" name="rviz"
        args="-d $(find hybrid_a)/rviz/display.rviz" />

    <node pkg="map_server" type="map_server" name="map_server" args="$(find hybrid_a)/maps/map.yaml"/>

    <!-- <node pkg="hybrid_a" type="hybrid_a_node" name="hybrid_a_node" > -->
    
    <node pkg="hybrid_a" type="hybrid_a_node" name="hybrid_a_node" output="screen">
        <param name="planner/steering_angle" value="17.0" />
        <param name="planner/steering_angle_discrete_num" value="1" />
        <param name="planner/wheel_base" value="2.0" />
        <param name="planner/segment_length" value="1.6" />
        <param name="planner/segment_length_discrete_num" value="8" />
        <param name="planner/steering_penalty" value="1.5" />
        <param name="planner/reversing_penalty" value="3.0" />
        <param name="planner/steering_change_penalty" value="2.0" />
        <param name="planner/shot_distance" value="500.0" />
        <param name="planner/length" value="4.7" />
        <param name="planner/width" value="2.0" />
        <param name="planner/rear" value="1.3" />

        <param name="traj_opti/vehicle/max_velocity" value="5" />
        <param name="traj_opti/vehicle/min_acceleration" value="-10.0" />
        <param name="traj_opti/vehicle/max_acceleration" value="10.0" />
        <param name="traj_opti/vehicle/jerk_max" value="5.0" />
        <param name="traj_opti/vehicle/phi_max" value="0.26" />
        <param name="traj_opti/vehicle/omega_max" value="2.0" />
        <rosparam file="$(find hybrid_a)/maps/paitu.yaml" command="load"/>
    </node>
</launch>